### ISSUE TITLE
not working

### ISSUE THREAD

**Issue by itfdw** [dlazaro66/QRCodeReaderView#93]
[](url)
![screenshot_1](https://cloud.githubusercontent.com/assets/20770526/20214572/feab94de-a806-11e6-8219-ee4c83051bc1.png)

**Comment by itfdw** [comment]
java.lang.NullPointerException
                                                                        at com.fdw.uxy.loreal.DecoderActivity.onQRCodeRead(DecoderActivity.java:44)
                                                                        at com.dlazaro66.qrcodereaderview.QRCodeReaderView$DecodeFrameTask.onPostExecute(QRCodeReaderView.java:347)
                                                                        at com.dlazaro66.qrcodereaderview.QRCodeReaderView$DecodeFrameTask.onPostExecute(QRCodeReaderView.java:301)

**Comment by itfdw** [comment]
@Override
    public void onQRCodeRead(String text, PointF[] points) {
        resultTextView.setText(text); --> error
    }

**Comment by itfdw** [comment]
Unable to resume activity 
  qrCodeReaderView.startCamera();

**Comment by itfdw** [comment]
my code

**Comment by dlazaro66** [comment]
Hey @itfdw You should create a local variable or a class field, up to you, but please name it as it should be, otherwise you're trying to assign the `findviewById(..)` result to a variable that is not declared (`mydecoderview`)

TL;DR:

![2016-11-11](https://cloud.githubusercontent.com/assets/3808535/20462949/02b72b50-af29-11e6-9686-b3aa837ebfb8.png)
